### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by ollita7**
Create new  angular environments:

qa -> logqa.cavepotlab.com (Running on EC2)
production -> log.cavepotlab.com (Running on FARGATE)

### CODE REVIEW

**Review by pabloferrari1789 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -35,3 +35,21 @@ Bellow you can see the available services
 * POST `/log/error` (creates a log entry with error level) [message, stacktrace]
 
 Every services needs to send the apikey of the client on the http headers
+
+### Docker
+```
```
Comment (santiagomussio) [inline README.md:40]: Maybe explain in readme the purpose of this.
For local deployment or for pushing into aws, or why is needed, or for what would i want to create a docker.
Maybe specify the folder in wich to run this commands

File: api/src/server.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ const options: IKiwiOptions = {
     middlewares: [HeadersMiddleware],
     cors: {
         enabled: true,
-        domains: ['http://log.cavepotlab.com', 'http://localhost:4200']
+        domains: ['http://log.cavepotlab.com', 'http://localhost:4200', 'https://log2.cavepotlab.com']
```
Comment (santiagomussio) [inline api/src/server.ts]: is log.cavepotlab used?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+version: '3'
+services:
+    migrate:
+        #image: public.ecr.aws/a4v3h0f0/vikingslog:v1
```
Comment (santiagomussio) [inline docker-compose.yml]: Explain in readme the difference for the commented image names and commented lines and how to use them

**Review by santiagomussio — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (ollita7) [reply docker-compose.yml]: removed because locally we are not using that!

File: api/src/server.ts (hunk 2 above)
Comment (ollita7) [reply api/src/server.ts]: No, removed

File: docker-compose.yml (hunk 3 above)
Comment (ollita7) [reply docker-compose.yml]: I will add logqa.cavepotlab.com

File: README.md (hunk 1 above)
Comment (ollita7) [reply README.md:40]: Not needed because locally we are not going to use docker.
Its only to remember for TC (When we have it I will remove it)

**Review by santiagomussio — APPROVED** [review]

**Pull request merged**